tests/integration/CartDiscount/CartDiscountUpdateRequestTest.php

```diff
@@ -357,23 +357,25 @@ function (CartDiscount $cartDiscount) use ($client) {
     public function testSetKey()
     {
         $client = $this->getApiClient();
+        $key = 'test-' . $this->getTestRun() . '-foo';
```

Comment:
not sure if should I leave "$this->getTestRun()" or replace with something else

Reply:
and if we replace it with something like ?
$description = Uuid::uuidv4() . 'new-description';

Reply:
If replacing it better use `'test-foo-'.Uuid::uuidv4()`

Reply:
Or directly use the CartDiscountFixture::uniqueCartDiscountString()